examples/anymcu-blink/platformio.ini

```diff
@@ -0,0 +1,13 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:n76e003]

```

Comment:
It looks like this example supports multiple boards (target devices). Does it make sense to declare multiple configuration environments to test them?